CONTRIBUTING.md

```diff
@@ -38,7 +38,8 @@ The format of each page should match the following:
 ```
 
 We actually have a linter/formatter that enforces our format.
-It even automatically cleans up your pages for you! Installing it is easy:
+It is run automatically on every pull request,
+but you may install it to test your contributions locally before submitting them:
 
 ```
 npm install
```

Comment:
Will it be better to write `npm test` instead of `tldr tldrl` ? The `npm test` was mentioned below but you removed it, which is fine. I think mentioning it here makes sense.